### PR TITLE
add initial archlinux support

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by dhoppe** [comment]
Are we really doing this before applying modulesync etc? By the way, you forgot to change metadata.json.

**Comment by bastelfreak** [comment]
I'm fine with waiting until your PR is merged.

**Comment by juniorsysadmin** [comment]
Needs unit tests

File: manifests/params.pp [hunk 1]
```diff
@@ -5,21 +5,31 @@
...
   $package_source      = undef
   $rpm_iteration       = '1'
-  $service_name        = 'grafana-server'
-  $version             = '2.5.0'
   $repo_name           = 'stable'
+  $version             = '2.5.0'
+  case $::osfamily {
+    'Archlinux': {
+      $manage_package_repo = false
+      $install_method      = 'repo'
```
Comment (bastelfreak) [inline manifests/params.pp]: there is a package in the arch repo and thats almost always up2date, so we can rely on it.

**Pull request closed without merging**